pkg/applicationserver/io/web/redis/registry.go

```diff
@@ -144,16 +146,19 @@ func (r WebhookRegistry) Set(ctx context.Context, ids ttnpb.ApplicationWebhookId
 				return nil
 			}
 		} else {
-			if pb.ApplicationWebhookIdentifiers != ids {
-				return errInvalidIdentifiers
-			}
-
 			pb.UpdatedAt = time.Now().UTC()
-			sets = append(sets, "updated_at")
+			sets = append(append(make([]string, 0, 2+len(sets)),
```

Comment:
Why `2+` ?

This level of micro-optimizations doesn't make sense really. To me, this has more downsides (i.e. cluttered code base) than benefits (saving some cpu cycles if Go doesn't optimize this already).